protocol/avdl/stellar1/common.avdl

```diff
@@ -22,6 +22,11 @@ protocol common {
     string limit;
   }
 
+  record AccountReserve {
+    string amount; // "-1"
+    string description; // e.g. "HUGZ trust line" or "account"
```

Comment:
HUGZ -> HUGZ/Unknown

Reply:
I mean.. that's what's in the designs, but whatever you think is fine.

Reply:
what makes a trustline "known"?

Reply:
Showing naked asset codes can be misleading. Here it's probably not a problem. Design seems to have elected for showing the federated name rather than the issuer account ID. But we don't resolve any federated asset names yet.